### ISSUE TITLE
why use logits instead of probabilities in computation of cost?

### ISSUE THREAD

**Issue by Michael-Jing** [udacity/deep-learning#42]
In dlnd_tv_script_generation.ipynb file, I noticed the following code snppet.

 `# Probabilities for generating words
    probs = tf.nn.softmax(logits, name='probs')

    # Loss function
    cost = seq2seq.sequence_loss(
        logits,
        targets,
        tf.ones([input_data_shape[0], input_data_shape[1]]))`

So I am confused why still use logits when probs is available?
